<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FMaquinasPage</class>
 <widget class="QRecordWidgetMapper" name="FMaquinasPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RecordWidgetMapper</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" rowspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="TTableViewModelRecord" name="tbvwMaquinas">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnNuevo">
         <property name="toolTip">
          <string>Nueva m치quina</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../app/app.qrc">
           <normaloff>:/pics/document-new.png</normaloff>:/pics/document-new.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAtras">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>M치quinas anteriores</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../app/app.qrc">
           <normaloff>:/pics/atras.png</normaloff>:/pics/atras.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAdelante">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Siguientes m치quinas</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../app/app.qrc">
           <normaloff>:/pics/adelante.png</normaloff>:/pics/adelante.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Droid Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Maquinas:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QGroupBox" name="grbxMaquina">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Maquina</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Direcci칩n IP:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="ledIp"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Lugar:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="TComboBoxModelRecord" name="cbxLugar"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>181</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="btnGuardar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
     <property name="icon">
      <iconset resource="../../app/app.qrc">
       <normaloff>:/pics/save.png</normaloff>:/pics/save.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btnCancelar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
     <property name="icon">
      <iconset resource="../../app/app.qrc">
       <normaloff>:/pics/dialog-close.png</normaloff>:/pics/dialog-close.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>84</width>
       <height>291</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TComboBoxModelRecord</class>
   <extends>QComboBox</extends>
   <header>tcomboboxmodelrecord.h</header>
  </customwidget>
  <customwidget>
   <class>TTableViewModelRecord</class>
   <extends>QTableView</extends>
   <header>ttableviewmodelrecord.h</header>
  </customwidget>
  <customwidget>
   <class>QRecordWidgetMapper</class>
   <extends>QWidget</extends>
   <header>qrecordwidgetmapper.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../app/app.qrc"/>
  <include location="../../app/app.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>FMaquinasPage</sender>
   <signal>hasRecordsWithOutSave(bool)</signal>
   <receiver>btnGuardar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FMaquinasPage</sender>
   <signal>hasRecordsWithOutSave(bool)</signal>
   <receiver>btnCancelar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>561</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbvwMaquinas</sender>
   <signal>navPrevious(bool)</signal>
   <receiver>btnAtras</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbvwMaquinas</sender>
   <signal>navNext(bool)</signal>
   <receiver>btnAdelante</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAtras</sender>
   <signal>clicked()</signal>
   <receiver>tbvwMaquinas</receiver>
   <slot>navToPrevious()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAdelante</sender>
   <signal>clicked()</signal>
   <receiver>tbvwMaquinas</receiver>
   <slot>navToNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbvwMaquinas</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>FMaquinasPage</receiver>
   <slot>toRecord(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
